library-managment-system/src/Details/SignInPage.js

```diff
@@ -0,0 +1,14 @@
+// src/pages/SignInPage.js
+import React from "react";
+import LoginForm from "../Components/LoginForm";
+import "../style.css";
+
+const SignInPage = ({ onSignIn }) => {
```

Comment:
Have not implemented sign out functionality

Reply:
Not yet. Do we need to implement the signet functionality as well ?